applications/DEMApplication/custom_constitutive/DEM_continuum_constitutive_law.cpp

```diff
@@ -50,6 +50,12 @@ namespace Kratos {
                 pProp->GetValue(DYNAMIC_FRICTION) = pProp->GetValue(FRICTION);
             }
         }
+        if(!pProp->Has(FRICTION_DECAY)) {
+            KRATOS_WARNING("DEM")<<std::endl;
+            KRATOS_WARNING("DEM")<<"WARNING: Variable FRICTION_DECAY should be present in the properties when using DEMContinuumConstitutiveLaw. 0.0 value assigned by default."<<std::endl;
+            KRATOS_WARNING("DEM")<<std::endl;
+            pProp->GetValue(FRICTION_DECAY) = 500.0;
```

Comment:
same
```suggestion
            pProp->GetValue(FRICTION_DECAY) = 0.0;
```

Reply:
Thanks! Finally is better to put 500 as a default value, I just changed it.